CHANGELOG.md

```diff
@@ -0,0 +1,97 @@
+
+  For a page `foo/page.md` Documenter now generates `foo/page/index.html`, instead of `foo/page.html`.
+  On GitHub pages deployments it means that your URLs look like  `foo/page/` instead of `foo/page.html`.
+
+  For local builds you should explicitly set `html_prettyurls = false`.
+
+  **For upgrading:** If you wish to retain the old behavior, set `html_prettyurls = false` in `makedocs`. If you already set `html_prettyurls`, you do not need to change anything.
+
+* ![BREAKING][badge-breaking] The `Travis.genkeys` and `Documenter.generate` functions have been moved to a separate [DocumenterTools.jl package](https://github.com/JuliaDocs/DocumenterTools.jl). ([#789][github-789])
+
+* ![Enhancement][badge-enhancement] If Documenter is not able to determine which Git hosting service is being used to host the source, the "Edit on XXX" links become "Edit source" with a generic icon. ([#804][github-804])
+
+* ![Enhancement][badge-enhancement] The at-blocks now support `MIME"text/html"` rendering of objects (e.g. for interactive plots). I.e. if a type has `show(io, ::MIME"text/html", x)` defined, Documenter now uses that when rendering the objects in the document. ([#764][github-764])
+
+* ![Enhancement][badge-enhancement] Minor style enhancements. ([#841][github-841])
+
+* ![Bugfix][badge-bugfix] The at-blocks that execute code can now handle `include` statements. ([#793][github-793], [#794][github-794])
+
+* ![Bugfix][badge-bugfix] At-docs blocks no longer give an error when containing empty lines. ([#823][github-823], [#824][github-824])
+
+[github-706]: https://github.com/JuliaDocs/Documenter.jl/pull/706
+[github-764]: https://github.com/JuliaDocs/Documenter.jl/pull/764
+[github-789]: https://github.com/JuliaDocs/Documenter.jl/pull/789
+[github-793]: https://github.com/JuliaDocs/Documenter.jl/pull/793
+[github-794]: https://github.com/JuliaDocs/Documenter.jl/pull/794
+[github-795]: https://github.com/JuliaDocs/Documenter.jl/pull/795
+[github-802]: https://github.com/JuliaDocs/Documenter.jl/pull/802
+[github-804]: https://github.com/JuliaDocs/Documenter.jl/pull/804
+[github-813]: https://github.com/JuliaDocs/Documenter.jl/pull/813
+[github-816]: https://github.com/JuliaDocs/Documenter.jl/pull/816
+[github-817]: https://github.com/JuliaDocs/Documenter.jl/pull/817
+[github-823]: https://github.com/JuliaDocs/Documenter.jl/pull/823
+[github-824]: https://github.com/JuliaDocs/Documenter.jl/pull/824
+[github-826]: https://github.com/JuliaDocs/Documenter.jl/pull/826
+[github-833]: https://github.com/JuliaDocs/Documenter.jl/pull/833
+[github-841]: https://github.com/JuliaDocs/Documenter.jl/pull/841
+
+
+[badge-breaking]: https://img.shields.io/badge/BREAKING-red.svg
+[badge-deprecation]: https://img.shields.io/badge/deprecation-red.svg
+[badge-feature]: https://img.shields.io/badge/feature-green.svg
+[badge-enhancement]: https://img.shields.io/badge/enhancement-blue.svg
+[badge-bugfix]: https://img.shields.io/badge/bugfix-purple.svg
```

Comment:
Love the badges. Selfmade or took the idea from somewhere else? We should use that in JuliaLang/julia.
Change deprecation to orange and bugfix to green?

Reply:
I synthesized the ideas from GitHub labels and the readme CI/doc badges I think. I am sure there is prior art though.

"Feature" is already green though. Could swap the colors though..